src/utils/session.ts

```diff
@@ -16,7 +15,16 @@ export function getPresenterAccountDetails(req: Request): Details | undefined {
 
 
 export function setPresenterAccountDetails(req: Request, details: Details) {
-    req.session?.setExtraData(PRESENTER_ACCOUNT_SESSION_KEY, details);
+    req.session?.setExtraData(ContextKeys.PRESENTER_ACCOUNT_SESSION_KEY, details);
+}
+
+export function getIsBusinessRegisteredFromExtraData(req: Request): boolean | undefined {
```

Comment:
I think we should include this variable in the presenter details json, keep all the session data in one place, as opposed to having it as a new extra data field.